### PR TITLE
Correctly set Content-Type for /_status response

### PR DESCRIPTION
**Pull request by mbyczkowski**
Without this change, `_/status` returns a JSON response with Content-Type set to `text/plain; charset=utf-8`

### CODE REVIEW

**Comment by csstaub** [comment]
Oooh we might need this fix in Ghostunnel too

File: server/server_test.go [hunk 1]
```diff
@@ -159,6 +159,42 @@ func TestGetKnownHosts(t *testing.T) {
...
+	rec := httptest.NewRecorder()
+	c.Status(rec, req)
+
+	rec.Flush()
+	if rec.Code != 200 {
+		t.Fatalf("Request to /_status failed with %d", rec.Code)
+	}
+
+	body, _ := ioutil.ReadAll(rec.Body)
+	expected, err := ioutil.ReadFile("testdata/status_ok.json")
```
Comment (mcpherrinm) [inline server/server_test.go]: could we just put this as a byte string here?  Seems silly given how short it is

Comment (mbyczkowski) [reply server/server_test.go]: yeah, that's fair!

File: server/server_test.go [hunk 2]
```diff
@@ -159,6 +159,41 @@ func TestGetKnownHosts(t *testing.T) {
...
+
+	rec.Flush()
+	if rec.Code != 200 {
+		t.Fatalf("Request to /_status failed with %d", rec.Code)
+	}
+
+	body, _ := ioutil.ReadAll(rec.Body)
+	expected := []byte(`{"ok":true,"status":"ok","messages":[]}`)
+	if err != nil {
+		t.Fatalf("Error reading testdata: %s", err)
```
Comment (mcpherrinm) [inline server/server_test.go]: stray err check, dead code

**Pull request merged**